src/client/nav/request.js

```diff
@@ -126,20 +126,20 @@ spf.nav.request.send = function(url, opt_options) {
       spf.config.get('experimental-request-headers'));
     if (configHeaders) {
       for (var key in configHeaders) {
-        headers[key] = configHeaders[key];
+        headers[key] = configHeaders[key] || '';  // Treat null, etc as empty.
```

Comment:
If the value is 0, it'll be overwritten to an empty string. Maybe check against not null/undefined explicitly or add a check for 0.